packit/patches.py

```diff
@@ -48,13 +48,15 @@ def __init__(
         description: Optional[str] = None,
         commit: Optional[git.Commit] = None,
         present_in_specfile: bool = False,
+        ignore: bool = False,
     ) -> None:
         self.name = name
         self.path = path
         self.location_in_specfile = location_in_specfile
         self.description = description
         self.commit = commit
         self.present_in_specfile = present_in_specfile
+        self.ignore = ignore
```

Comment:
please document the expected behaviour here, 'ignore' can mean a lot of things

Reply:
Added docstring. PTAL